### PR TITLE
WD-2765 Build the 18-04 out of service page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done

- Build the 18-04 out-of-service page from the [copy doc](https://docs.google.com/document/d/1gbygbkkqvWJ-PbkOfTpeL-n8sKlgY-88F_4CNbIPqJA/edit#)

## QA

- Open the demo and go to /18-04
- Check it matches the design: https://www.figma.com/file/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-Ubuntu-Pro?node-id=3628-18766&t=1PbS0G37UUwWbRvh-0
- Check the content matches the copy doc above
- Check different screen sizes

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
@anthonydillon sorry what is the url of the page?

**Comment by anthonydillon** [comment]
> @anthonydillon sorry what is the url of the page?
> 

https://ubuntu-com-12711.demos.haus/18-04

File: templates/18-04/index.html [hunk 1]
```diff
@@ -0,0 +1,214 @@
+{% extends "18-04/base_18-04.html" %}
+
+{% block title %}Ubuntu 18.04 LTS (Bionic Beaver){% endblock %}
+{% block meta_description %}Ubuntu 18.04 LTS (Bionic Beaver) is out of standard support on 31 May 2023. Ubuntu Pro provides continued support and security coverage for Ubuntu 18.04 LTS for additional 5 years, until 2028.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1gbygbkkqvWJ-PbkOfTpeL-n8sKlgY-88F_4CNbIPqJA/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip">
+  <div class="row">
+    <div class="col-5 u-hide--small u-hide--medium">
```
Comment (lyubomir-popov) [inline templates/18-04/index.html:10]:
```suggestion
    <div class="col-5 col-medium-3 u-hide--small">
```

File: templates/18-04/index.html [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+      <img src="https://assets.ubuntu.com/v1/bb580eec-Bionic+Beaver_18.04.svg" alt="" />
+    </div>
+    <div class="col-6 col-start-large-7">
```
Comment (lyubomir-popov) [inline templates/18-04/index.html:13]:
```suggestion
    <div class="col-medium-3col-6 col-start-large-7">
```

File: templates/18-04/index.html [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+      <h1>Ubuntu 18.04 LTS</h1>
```
Comment (lyubomir-popov) [inline templates/18-04/index.html:14]:
```suggestion
      <h1 class="p-heading--2"><strong>Ubuntu 18.04 LTS</strong></h1>
```

File: templates/18-04/index.html [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+      <p class="p-heading--2">Out of standard support. <br class="u-hide--small">Upgrade to Ubuntu Pro</p>
+      <p>Ubuntu 18.04 LTS (Bionic Beaver) was released on 23 April 2018, introducing improved UEFI Secure Boot, broader Kernel Livepatch coverage, GNOME instead of Unity on Ubuntu Desktop, LXD 3.0 and OpenStack Queens.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <hr class="u-fixed-width" />
+  <div class="row">
+    <div class="col-6">
```
Comment (lyubomir-popov) [inline templates/18-04/index.html:24]:
```suggestion
    <div class="col-6 col-medium-3">
```

File: templates/18-04/index.html [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+      <p>Transitioning to the latest operating system, such as Ubuntu 22.04 LTS, is important for performance, hardware enablement and new technology benefits and is recommended for new instances. But it might be a more complex process for existing deployments.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <hr class="u-fixed-width" />
+  <div class="row">
+    <div class="col-6">
```
Comment (lyubomir-popov) [inline templates/18-04/index.html:36]:
```suggestion
    <div class="col-6 col-medium-3">
```

File: templates/18-04/index.html [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+      <h3 class="p-heading--2">What are your options?</h3>
+    </div>
+    <div class="col-6">
```
Comment (lyubomir-popov) [inline templates/18-04/index.html:39]:
```suggestion
    <div class="col-6 col-medium-3">
```

File: templates/18-04/index.html [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+      <p><a href="/pro" class="p-button--positive">Get started with Ubuntu Pro for 18.04 LTS</a></p>
+      <p><a href="/cloud/public-cloud">Learn more about Ubuntu Pro on public clouds&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <hr class="u-fixed-width" />
+  <div class="row">
+    <div class="col-6">
```
Comment (lyubomir-popov) [inline templates/18-04/index.html:51]:
```suggestion
    <div class="col-6 col-medium-3">
```

**Review by lyubomir-popov — COMMENTED** [review]
A few suggestions in the comments. I wou7ld add col-medium-3 on all col-6 divs. And I've corrected u-fixed-width to is-fixed-width on hrs, otherwise the hr goes into the grid padding which we don't want.

**Comment by anthonydillon** [comment]
@lyubomir-popov address you comments. Why only some of the heading get the col-medium-3 treatment?

**Comment by lyubomir-popov** [comment]
> @lyubomir-popov address your comments. Why do only some of the headings get the col-medium-3 treatment?

All of them should, I just did the first few as an example of what needs to happen everywhere.

File: templates/18-04/index.html [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+        <a href="/engage/case-study-acuris">Read the case study&nbsp;&rsaquo;</a>
+      </blockquote>
+      <hr />
+      <p><a href="/security/esm#get-in-touch">Get in touch</a> if you need advice on the best path for your company.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-deep">
+  <div class="row">
```
Comment (mtruj013) [inline templates/18-04/index.html:209]: From the design, looks like there should be an hr above this row to separate this section

**Review by mtruj013 — APPROVED** [review]
LGTM! Just 2 small comments:

**Comment by anthonydillon** [comment]
@mtruj013 thanks done

**Pull request merged**